src/Chapter1.hs

```diff
@@ -119,125 +99,92 @@ immediately see what types will be inferred.
 |  Parsing  +------> Compilation +-----> Executable  |
 |           |      |             |     |             |
 +-----------+      +-------------+     +-------------+
-
 In comparison, such languages as Python, JavaScript, etc. are interpreted
 languages. And that could be illustrated in the different workflow:
-
 +-----------+      +----------------+     +------------+
 |           |      |                |     |            |
 |  Parsing  +------> Interpretation +-----> Evaluation |
 |           |      |                |     |            |
 +-----------+      +----------------+     +------------+
-
 So, when working with Haskell, you first need to compile the code in order to
 run it later.
-
 However, most of the time while working with Haskell (especially when learning
 Haskell), you use __GHCi__ to play with functions, see their behaviour and
 explore API of external libraries. GHCi is a __Haskell REPL__
 (read-eval-print-loop). It allows calling functions directly in an interactive
 shell. GHCi interprets Haskell expressions and statements and prints the
 evaluation result of each expression.
-
 As you can see, Haskell supports both paradigms: you can either compile your
 code and run the executable, or interpret the code and run it directly.
-
 Assuming that you already have the Haskell compiler installed (we recommend
 GHC), you can start GHCi by executing the following command in your terminal
 from the root of this project.
-
 $ ghci
-
 > If you don't have Haskell yet, refer to the corresponding section of the
   README.
-
 Now, you can evaluate some expressions and see their results immediately.
-
 >>> 1 + 2
 3
-
 ♫ NOTE: Each time you see a line starting with >>> in this training, it means
   that the text after >>> is supposed to be evaluated in GHCi, and the evaluation
   result should be printed on the next line if it has any. In your GHCi, you
   probably have the string "Prelude> " as your prompt. For some expressions, we
   already prefilled the result, but for others, you need to insert it on your own.
   We use lovely Haskell tools to test the results of the GHCi evaluation as well,
   so you'd better insert exact answers in order to make them pass the testing!
-
   You will also see lines started with "ghci>". They are also supposed to be
   evaluated in GHCi, but our testing system doesn't check their output.
   They are here just to showcase the different usages of GHCi.
-
-
 GHCi can do much more than evaluating expressions. It also contains some special
 commands starting with a colon. For example, to see the list of all available
 commands, type ":?" in your GHCi.
-
 ghci> :?
-
 To quit GHCi, enter the ":q" command (short for ":quit").
-
 ghci> :q
-
 -}
 
 {- |
 =⚔️= Task 1
-
 Since types play a crucial role in Haskell, we can start by exploring types of
 some basic expressions. You can inspect the type of expression by using the ":t"
 command in GHCi (short for ":type").
-
 For example:
-
 >>> :t False
 False :: Bool
-
```

Comment:
Unfortunately, removing newlines from such examples, and exercises below, will not allow tests to pass. As we mention, we use a special testing system for such examples 🙂